docs/_includes/ds6/dialog.html

```diff
@@ -0,0 +1,489 @@
+<div id="dialog">
+  <h2><span class="secondary-text">@ebay/skin/</span>dialog</h2>
+  <p>A dialog is a child window spawned by the main web page or application.</p>
+  <p>This window forms a new background layer in our design system topography.</p>
+  <p>A dialog must remain in a
+    <span class="highlight">hidden</span> state for all users and devices until called upon.</p>
+  <p>All dialogs have support for
+    <em>optional</em> CSS transition effects. See
+    <a href="#dialog-transitions">dialog transitions</a> section for more details.</p>
+
+  <h3>Default Dialog</h3>
+  <p>The default dialog is a lightbox.</p>
+  <p>The lightbox will have a margin of 15vh and automatically scales vertically.</p>
+  <div class="demo">
+    <div class="demo__inner">
+      <button class="btn btn--primary dialog-button" type="button">Preview</button>
+      <div role="dialog" class="dialog" hidden>
+        <div class="dialog__mask" role="button"></div>
+        <div class="dialog__window" role="document">
+          <button aria-label="Close dialog" class="dialog__close" type="button">
```

Comment:
The reason I went with the inline sag over background image is because on hover the color of the svg changes. I'd actually have to create two inline background svgs to achieve this.

Just want to be sure that's the route we want to go.

Reply:
Yeah, for maximum flexibility, we are going to support both moving forward (using `dialog__close > svg` (just positions the svg correctly without knowing exactly which icon is being used) or `dialog__close > span` (uses the two background svgs, ideally from ds6 icon mixins).